nvflare/private/fed/app/server/runner_process.py

```diff
@@ -129,6 +129,7 @@ def start_server_app(server, args, app_root, run_number, snapshot):
         if not isinstance(server.engine, ServerEngine):
             raise TypeError(f"server.engine must be ServerEngine. Got type:{type(server.engine).__name__}")
         server.engine.create_parent_connection(int(args.conn))
+        server.engine.sync_clients_from_main_process()
```

Comment:
What does sync_clients_from_main_process do? Does it only keep clients applicable to the job?

Reply:
Yes, sync_clients_from_main_process only keep the clients running the job.